analyzer/src/test/kotlin/managers/CarthageTest.kt

```diff
@@ -51,6 +51,7 @@ class CarthageTest : WordSpec() {
                         id.type shouldBe "Carthage"
                         vcs.url shouldBe "https://github.com/Alamofire/AlamofireImage.git"
                         vcs.revision shouldBe "3.2.0"
+                        copyrightHolders shouldBe emptySet()
```

Comment:
Please drop the changes to this file. The `authors` aren't asserted here either. I guess felt urged to add these to increase code coverage, this is not required as discussed.

Reply:
I would then prefer to assert the authors as well. Enhances the test coverage.